+++
title = "MiRANN: A reliable approach for improved classification of precursor microRNA using Artificial Neural Network model"
date = "2012-02-10"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Md. Eamin Rahman", "Rashedul Islam", "Shahidul Islam", "Shakhinur Islam Mondal", "Ruhul Amin"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["1"]

# Publication name and optional abbreviated version.
publication = "Genomics, Volume 99, Issue 4"
publication_short = "Genomics"

# {{< figure src="updated_pipeline.jpg" title="Steve Francia" >}}

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
# image = "updated_pipeline.jpg"
# [header]
# image = "updated_pipeline.jpg"
# caption = "Genome Annotation with Deep Learning"

# Abstract and optional shortened version.
abstract = "MicroRNA (miRNA) is a special class of short noncoding RNA that serves pivotal function of regulating gene expression. The computational prediction of new miRNA candidates involves various methods such as learning methods and methods using expression data. This article has proposed a reliable model â€” miRANN which is a supervised machine learning approach. MiRANN used known pre-miRNAs as positive set and a novel negative set from human CDS regions. The number of known miRNAs is now huge and diversified that could cover almost all characteristics of unknown miRNAs which increases the quality of the result (99.9% accuracy, 99.8% sensitivity, 100% specificity) and provides a more reliable prediction. MiRANN performs better than other state-of-the-art approaches and declares to be the most potential tool to predict novel miRNAs. We have also tested our result using a previous negative set. MiRANN, opens new ground using ANN for predicting pre-miRNAs with a promise of better performance."

# abstract_short = "Genome annotation is the process of labeling DNA sequences of an organism with its biological features, and is one of the fundamental problems in Bioinformatics. In this paper, we evaluate DNA K-mer embeddings and the application of RNNs for genome annotation. We show how to improve the performance of our deep networks by incorporating intermediate objectives and downstream algorithms to achieve better accuracy."

# Is this a selected publication? (true/false)
selected = true

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["language variation"]

# Links (optional).
# url_pdf = "http://arxiv.org/pdf/1512.04133v1"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_pdf = "https://www.sciencedirect.com/science/article/pii/S0888754312000262"


# Featured image thumbnail (optional)
image_preview = ""

[header]
image = ""
caption = ""


+++

