+++
title = "Re-annotation of 12,495 prokaryotic 16S rRNA 3’ ends and analysis of Shine-Dalgarno and anti-Shine-Dalgarno sequences"
date = "2018-08-23"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Ruhul Amin", "Alisa Yurovsky", "Yuping Chen", "Steven Skiena", "Bruce Futcher"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["1"]

# Publication name and optional abbreviated version.
publication = "PLOS One, Volume 13, Issue 8"
publication_short = "PLOS One"

# {{< figure src="updated_pipeline.jpg" title="Steve Francia" >}}

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
# image = "updated_pipeline.jpg"
# [header]
# image = "updated_pipeline.jpg"
# caption = "Genome Annotation with Deep Learning"

# Abstract and optional shortened version.
abstract = "We examined 20,648 prokaryotic unique taxids with respect to the annotation of the 3’ end of the 16S rRNA, which contains the anti-Shine-Dalgarno sequence. We used the sequence of highly conserved helix 45 of the 16S rRNA as a guide. By this criterion, 8,153 annotated 3’ ends correctly included the anti-Shine-Dalgarno sequence, but 12,495 were foreshortened or otherwise mis-annotated, missing part or all of the anti-Shine-Dalgarno sequence, which immediately follows helix 45. We re-annotated, giving a total of 20,648 16S rRNA 3’ ends. The vast majority indeed contained a consensus anti-Shine-Dalgarno sequence, embedded in a highly conserved 13 base “tail”. However, 128 exceptional organisms had either a variant anti-Shine-Dalgarno, or no recognizable anti-Shine-Dalgarno, in their 16S rRNA(s). For organisms both with and without an anti-Shine-Dalgarno, we identified the Shine-Dalgarno motifs actually enriched in front of each organism’s open reading frames. This showed to what extent the Shine-Dalgarno motifs correlated with anti-Shine Dalgarno motifs. In general, organisms whose rRNAs lacked a perfect anti-Shine-Dalgarno motif also lacked a recognizable Shine-Dalgarno. For organisms whose 16S rRNAs contained a perfect anti-Shine-Dalgarno motif, a variety of results were obtained. We found one genus, Alteromonas, where several taxids apparently maintain two different types of 16S rRNA genes, with different, but conserved, antiSDs. The fact that some organisms do not seem to have or use Shine-Dalgarno motifs supports the idea that prokaryotes have other robust mechanisms for recognizing start codons for translation."

# abstract_short = "Genome annotation is the process of labeling DNA sequences of an organism with its biological features, and is one of the fundamental problems in Bioinformatics. In this paper, we evaluate DNA K-mer embeddings and the application of RNNs for genome annotation. We show how to improve the performance of our deep networks by incorporating intermediate objectives and downstream algorithms to achieve better accuracy."

# Is this a selected publication? (true/false)
selected = true

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["language variation"]

# Links (optional).
# url_pdf = "http://arxiv.org/pdf/1512.04133v1"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = "https://sites.google.com/cs.stonybrook.edu/16s"
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_pdf = "https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0202767"


# Featured image thumbnail (optional)
# image_preview = "rrna_correction.png"

# [header]
# image = "rrna_correction.png"
# caption = "Weblogo of Corrected and Uncorrected 16S 3' end"


+++

