+++
title = "Prokaryotic coding regions have little if any specific depletion of Shine-Dalgarno motifs"
date = "2018-08-23"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Alisa Yurovsky", "Ruhul Amin", "Justin Gardin", "Yuping Chen", "Steven Skiena", "Bruce Futcher"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["1"]

# Publication name and optional abbreviated version.
publication = "PLOS One, Volume 13, Issue 8"
publication_short = "PLOS One"

# {{< figure src="updated_pipeline.jpg" title="Steve Francia" >}}

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
# image = "updated_pipeline.jpg"
# [header]
# image = "updated_pipeline.jpg"
# caption = "Genome Annotation with Deep Learning"

# Abstract and optional shortened version.
abstract = "The Shine-Dalgarno motif occurs in front of prokaryotic start codons, and is complementary to the 3’ end of the 16S ribosomal RNA. Hybridization between the Shine-Dalgarno sequence and the anti-Shine-Dalgarno region of the16S rRNA (CCUCCU) directs the ribosome to the start AUG of the mRNA for translation. Shine-Dalgarno-like motifs (AGGAGG in E. coli) are depleted from open reading frames of most prokaryotes. This may be because hybridization of the 16S rRNA at Shine-Dalgarnos inside genes would slow translation or induce internal initiation. However, we analyzed 128 species from diverse phyla where the 16S rRNA gene(s) lack the anti-Shine-Dalgarno sequence, and so the 16S rRNA is incapable of interacting with Shine-Dalgarno-like sequences. Despite this lack of an anti-Shine-Dalgarno, half of these species still displayed depletion of Shine-Dalgarno-like sequences when analyzed by previous methods. Depletion of the same G-rich sequences was seen by these methods even in eukaryotes, which do not use the Shine-Dalgarno mechanism. We suggest previous methods are partly detecting a non-specific depletion of G-rich sequences. Alternative informatics approaches show that most prokaryotes have only slight, if any, specific depletion of Shine-Dalgarno-like sequences from open reading frames. Together with recent evidence that ribosomes do not pause at ORF-internal Shine-Dalgarno motifs, these results suggest the presence of ORF-internal Shine-Dalgarno-like motifs may be inconsequential, perhaps because internal regions of prokaryotic mRNAs may be structurally “shielded” from translation initiation."

# abstract_short = "Genome annotation is the process of labeling DNA sequences of an organism with its biological features, and is one of the fundamental problems in Bioinformatics. In this paper, we evaluate DNA K-mer embeddings and the application of RNNs for genome annotation. We show how to improve the performance of our deep networks by incorporating intermediate objectives and downstream algorithms to achieve better accuracy."

# Is this a selected publication? (true/false)
selected = true

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["language variation"]

# Links (optional).
# url_pdf = "http://arxiv.org/pdf/1512.04133v1"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = "https://sites.google.com/cs.stonybrook.edu/16s"
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_pdf = "https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0202767"


# Featured image thumbnail (optional)
# image_preview = "ribosome_evoluation.png"

# [header]
# image = "ribosome_evolution.png"
# caption = "Evolution of ribosomal proteins S1 and S10 in organisms lacking an antiSD"


+++

