+++
title = "SHMPrep: A Fast and User-Friendly Program for Preprocessing Paired-end Immunoglobulin Sequence Data from Illumina Platforms"
date = "2019-01-31"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Ruhul Amin", "Jeewoen Shin", "Sergio Roa", "Jeffery Chen", "Anne Jeannin-Girardon", "Stephen Meier", "Thomas Maccarthy"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["3"]

# Publication name and optional abbreviated version.
publication = "Nucleic Acids Research"
publication_short = "NAR"

# {{< figure src="updated_pipeline.jpg" title="Steve Francia" >}}

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
# image = "updated_pipeline.jpg"
# [header]
# image = "updated_pipeline.jpg"
# caption = "Genome Annotation with Deep Learning"

# Abstract and optional shortened version.
abstract = "Next generation sequencing platforms, predominantly the Illumina MiSeq, are now routinely used to sequence immunoglobulin (Ig) genes to measure effects such as somatic hypermutation (SHM). There is a need for software that can be used widely by experimentalists who may not have computational expertise or access to specialized computers. We have developed SHMPrep, a very fast and user-friendly program for preprocessing paired-end immunoglobulin data from the Illumina MiSeq. The program aligns paired-end reads, performs error correction and collapses sequences. Implementation of efficient string matching algorithms such as suffix automata and string alignment algorithms such as k-band global alignment, enable processing of an entire MiSeq run of ~20 million reads in approximately 30 minutes using a standard desktop computer. SHMPrep can also handle barcodes (UMI) and multiple sets of primers, making it ideal for working with Ig repertoire data. The program produces outputs in FASTA/Q format that can be further processed using tools such as IMGT. The software is composed of a kernel written in the high-performance C++ language and a user-friendly Graphical User Interface (GUI) developed in Java that is designed for use by non-computational experts. The software is available for Windows, Mac and Linux. The software is available at: http://www.ams.sunysb.edu/~maccarth/software.html"

# abstract_short = "Genome annotation is the process of labeling DNA sequences of an organism with its biological features, and is one of the fundamental problems in Bioinformatics. In this paper, we evaluate DNA K-mer embeddings and the application of RNNs for genome annotation. We show how to improve the performance of our deep networks by incorporating intermediate objectives and downstream algorithms to achieve better accuracy."

# Is this a selected publication? (true/false)
selected = true

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["language variation"]

# Links (optional).
# url_pdf = "http://arxiv.org/pdf/1512.04133v1"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_pdf = ""


# Featured image thumbnail (optional)
image_preview = ""

[header]
image = ""
caption = ""


+++

